composer.json

```diff
@@ -240,5 +242,12 @@
             "@sa:check",
             "@lint:check"
         ]
+    },
+    "extra": {
+        "patches": {
+            "doctrine/dbal": {
+                "Doctrine migrations functional index workaround": "dbal-funcational-index.patch.txt"
+            }
+        }
```

Comment:
Having copied the method of indexing out of identity and then found the need to patch dbal, I copied this patch back into identity in https://github.com/thebiggive/identity/pull/193